x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/type/DataTypes.java

```diff
@@ -48,8 +48,6 @@ public final class DataTypes {
     public static final DataType IP               = new DataType("ip",                45,                false, false, true);
     // version
     public static final DataType VERSION          = new DataType("version",           Integer.MAX_VALUE, false, false, true);
-    // binary
-    public static final DataType BINARY           = new DataType("binary",            Integer.MAX_VALUE, false, false, true);
```

Comment:
Why are we removing Binary? is it just never used in ESQL?

Reply:
Yeah, never used at all.